src/main/java/trade/wayruha/whitebit/dto/Asset.java

```diff
@@ -0,0 +1,49 @@
+package trade.wayruha.whitebit.dto;
+
+import lombok.Data;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class Asset {
+    private String name;
+    private BigDecimal unified_cryptoasset_id;
+    private boolean can_withdraw;
```

Comment:
please rename variables to follow camelCase pattern. e.g., canWithdraw.
In order to deserialize it correctly, we would need to annotate each such field with its deserialization name using @JsonAlias("can_withdraw")

Reply:
fixed